src/Cli.Tests/Snapshots/InitTests.CosmosDbPostgreSqlDatabase.verified.txt

```diff
@@ -5,7 +5,8 @@
   Runtime: {
     Rest: {
       Enabled: true,
-      Path: /rest-endpoint
+      Path: /rest-endpoint,
```

Comment:
not sure which test uses this file, but this is example of custom rest path set and written to config for cosmos.

Reply:
Right, this is cosmosdb for pg and hence it's written to the config.